core/src/util/builders.cpp

```diff
@@ -137,6 +137,12 @@ void Builders::buildPolygonExtrusion(const Polygon& _polygon, float _minHeight,
             normalVector = glm::cross(upVector, b - a);
             normalVector = glm::normalize(normalVector);
 
+            if (std::isnan(normalVector.x)
```

Comment:
This is good, however I think we should be catching degenerate points within a valid polygon ring before we get to this stage!